src/core/util/util.js

```diff
@@ -40,132 +40,97 @@ angular.module('material.core')
       return Util.clientRect(element, offsetParent, true);
     },
     disableScrollAround: function(element) {
```

Comment:
While this is fresh in your mind, can you add a function comment that explains what the function does and, at a high level, _how_ it does it?